### PR TITLE
Update the signature x5u cache bust to be configurable.

### PR DESCRIPTION
**Pull request by Osmose**
Turns out it'll cost a fair amount of CDN money to re-serve this to users until we'd need to disable it, so let's just make it static and configurable.

### CODE REVIEW

File: recipe-server/normandy/settings.py [hunk 1]
```diff
@@ -321,6 +321,7 @@ def RAVEN_CONFIG(self):
     AUTOGRAPH_HAWK_ID = values.Value()
     AUTOGRAPH_HAWK_SECRET_KEY = values.Value()
     AUTOGRAPH_SIGNATURE_MAX_AGE = values.IntegerValue(60 * 60 * 24 * 7)
+    AUTOGRAPH_X5U_CACHE_BUST = values.Value(None)
```
Comment (mythmon) [inline recipe-server/normandy/settings.py]: This setting needs documented.

File: recipe-server/normandy/recipes/tests/api/v1/test_serializers.py [hunk 2]
```diff
@@ -240,12 +240,21 @@ def test_it_works(self):
...
+        assert 'cachebust' not in query
 
+        # If set, cachebust using the value
+        settings.AUTOGRAPH_X5U_CACHE_BUST = 'new'
+        serializer = SignatureSerializer(instance=signature)
         url_parts = list(urlparse.urlparse(serializer.data['x5u']))
         query = urlparse.parse_qs(url_parts[4])
         assert 'cachebust' in query
         assert len(query['cachebust']) == 1
-        assert re.match('^\d+$', query['cachebust'][0])
```
Comment (mythmon) [inline recipe-server/normandy/recipes/tests/api/v1/test_serializers.py]: The linter says `recipe-server/normandy/recipes/tests/api/v1/test_serializers.py:1:1: F401 're' imported but unused` now.

**Comment by Osmose** [comment]
Updated.

**Comment by mythmon** [comment]
The TC issue is #806.

**Pull request merged**